### PR TITLE
chore: bump to 2.0

### PR DESCRIPTION
**Pull request by CatStark**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM jinaai/jina:master as base
+FROM jinaai/jina:2.0
```
Comment (nan-wang) [inline Dockerfile:1]: Delete line8-12 to remove the tests. There is no tests running on the hubble.

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,6 @@
...
-RUN pip install .
-
-FROM base
-RUN pip install -r tests/requirements.txt
-RUN pytest -s -v tests
-
-FROM base
-ENTRYPOINT ["jina", "executor", "--uses", "config.yml"]
\ No newline at end of file
+RUN pip install .
```
Comment (cristianmtr) [inline Dockerfile:6]: You need the entrypoint, right?

Comment (CatStark) [reply Dockerfile:6]: Yep, my bad

**Review by jakob1111996 — APPROVED** [review]

**Review by nan-wang — APPROVED** [review]
LGTM👍

**Pull request merged**